app/parse_class_processing.py

```diff
@@ -0,0 +1,155 @@
+import csv
+import threading
+import time
+import requests
+import multiprocessing
+
+from dataclasses import dataclass, fields, astuple
+from bs4 import BeautifulSoup
+from urllib.parse import urljoin
+from concurrent.futures import ProcessPoolExecutor, wait
+
+
+@dataclass
+class Quote:
```

Comment:
Lots of code duplicates in 3 files. Fix them

Reply:
I know, fix this after fix parallel run code in parse_class_processin, ok?))

Reply:
Can you see the code and help fix the errors?